docs/release-notes/8.8.asciidoc

```diff
@@ -18,14 +18,15 @@ To view a detailed summary of new features and enhancements we've documented thi
 
 //tag::breaking-changes[]
 // NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+// THIS ALSO MEANS IF YOU USE LINKS HERE, THEY SHOULD BE HARDCODED UNLESS ANY ATTRIBUTES ARE ALSO DEFINED WHERE THE SNIPPET IS REFERENCES
```

Comment:
It might help to be specific about what "hardcoded" means. Also, I'd rather just not even give locally-defined attributes as an option; it complicates things and in most cases not necessary — we can use a full URL.
```suggestion
// THIS ALSO MEANS IF YOU USE LINKS HERE, THEY SHOULD BE FULL URLS WITH NO ATTRIBUTES
```